src/Core/Checkout/Customer/Event/WishlistProductAddedEvent.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Shopware\Core\Checkout\Customer\Event;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\ShopwareSalesChannelEvent;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class WishlistProductAddedEvent implements ShopwareSalesChannelEvent
+{
+    /**
+     * @var string
```

Comment:
there is no need for doc annotations if they contain the same information as the native, so please remove all doc annotations for the typed properties, typed params and typed returns in this PR

Reply:
Thanks for the guidelines. The corrections have been sent. Do you need to create tests or add information to the documentation for this type of change?

Reply:
TestCases would be great, i think you can add testCases in the existing route tests:

`\Shopware\Core\Checkout\Test\Customer\SalesChannel\AddWishlistProductRouteTest`
`\Shopware\Core\Checkout\Test\Customer\SalesChannel\RemoveWishlistProductRouteTest`

and assert that the right events are thrown by adding a listener to the event dispatcher that asserts that the event is thrown and has the correct data, something similiar is already done here `\Shopware\Core\Checkout\Test\Customer\Api\CustomerGroupRegistrationActionControllerTest::testEventDispatchCustomerRegistrationAcceptedEventCorrectLanguage`